src/main/java/util/DataGeneration.java

```diff
@@ -0,0 +1,16 @@
+package util;
+
+public class DataGeneration {
+
+    public String generateUniqueTradeName() {
```

Comment:
it could be more unified

public static generateRandomString(String addValue, int quantityOfNumers){
return addValue + (int) (Math.random() * 1*quantityOfNumers);
}